common/scripted_triggers/wc_portrait_triggers.txt

```diff
@@ -29,27 +29,14 @@ has_no_clothes_trigger = {
 prefer_no_lowborn_headgear = {
 	OR = {
 		# Ears get in the way
-		culture = { has_graphical_culture = no_headgear_gfx }
+		culture = { has_graphical_culture = no_lowborn_headgear_gfx }
```

Comment:
Looks like you based it on the dwarven branch.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/152086567-bd98b383-54a2-44a5-8ab5-70b1d4c507ff.png)

</details>
Edit the base branch to the dwarven one here.